### PR TITLE
Adds 2 posters

### PR DESCRIPTION
**Pull request by IamVelcroboy**
Changelog: 
-Adds No Touching poster for prison visitation rooms
-Adds Medicate poster for Therapist/Chem/Medical lobbies

  
Screenshot 2022-12-30 175747

### CODE REVIEW

**Comment by Elijahrane** [comment]
need to fix up the RSI so it passes tests

**Comment by IamVelcroboy** [comment]
Honestly, I can't find anything wrong with it.

**Comment by IamVelcroboy** [comment]
I think it's just failing on some random pngs that aren't defined in an rsi file. It doesn't even mention these ones.

**Comment by Vordenburg** [comment]
It was failing due to issues unrelated to this PR.

**Comment by ZeroDayDaemon** [comment]
I have added them to this PR, because I do not respect git

**Comment by ZeroDayDaemon** [comment]
Aw fuck me I fix one and another completely unrelated one appears

File: Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml [hunk 1]
```diff
@@ -107,3 +107,23 @@
...
...ite, irresistible ability to organize bookshelves either by color or alphabetically, x-ray vision, slurred speech, impromptu levitation, inability to communicate in any other way besides pig Latin, incontinence, sudden uncontrollable urge to scream “FUCK FUCK FUCK” at the top of your lungs, unanticipated grasp of juggling, insomnia, narcolepsy, leprosy, spontaneous detachment of fingers or toes, unstoppable back hair growth, weight loss, weight gain, shrinkage, colorblindness, tingling in earlobes, rectal numbness, acute ability to taste colors, unquenchable bloodlust, restless fingernails, unappeasable hunger for fruitcake, fierce devotion to K-pop, capability to dream in Chinese, inability to forgive others, unceremonious LARPing, puffy elbows, sudden onset of ability to yodel, light-sensitivity, compulsion to wear kilts, talent for witchcraft, scurvy, eagerness to pursue mercenary life, unrestrained yearning to breakdance, violent craving to skateboard naked, insurgent desire to develop a gambling addiction, uncompromising aspiration to join a flash mob, itchy butt. Call your doctor for medical advice about these or any other side effects you may experience." #credit Chris Zappa
```
Comment (Vordenburg) [inline Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: Do you have a screenshot of what this long string of text looks like in game?

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: Oh, I never saw this....
😂
image

Comment (Vordenburg) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: It looks about as big as I expected it to be. I feel like this is too long for an item description. Other than that issue, this is able to be merged.

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: I mean, that was sort of the point of the joke...It kind of should be absurdly long, no?

Comment (OldDanceJacket) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: Yes but it's a bit **too** long, I think keeping the joke but trimming down the size could be good, think the length where it gets to FUCK FUCK FUCK is still obnoxiously long without it being excessive.

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: 😩Well,  I think it's about 60% less funny/impactful but...is this what you'd prefer? 
image

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/posters.yml]: Also, don't take my melodrama too seriously. I'm just a fucking dork sometimes. I think this is fine if you're ok with it.

**Comment by Elijahrane** [comment]
add to poster spawners...?

**Comment by Vordenburg** [comment]
I think the idea is that they're for specific areas.

**Comment by Elijahrane** [comment]
OK

**Pull request merged**